MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt

```diff
@@ -133,11 +134,17 @@ class TelemetrySyncManager(
         val totalTelemetrySize = rootTelemetryDirectory.directorySizeRecursive()
         if (totalTelemetrySize > MAX_TELEMETRY_SIZE) {
             var bytesToRemove = totalTelemetrySize - MAX_TELEMETRY_SIZE
-            val sortedTelemetryDirs = rootTelemetryDirectory
-                .listFiles()
-                .sortedBy { it.name }
+            val telemetryDirs = mutableListOf<File>()
 
-            for (telemetryDir in sortedTelemetryDirs) {
+            val countryDirs = rootTelemetryDirectory.listFiles()
+            countryDirs?.forEach { dir ->
```

Comment:
maybe change _dir_ to _countryDir_, _it_ -> _timestampDir_

Reply:
agree, done